pyproject.toml

```diff
@@ -21,7 +21,7 @@ keywords = ["api", "graphql", "protocol", "rest", "relay", "graphene", "pydantic
 [tool.poetry.dependencies]
 python = "^3.6"
 # To keep things simple, we only support newer versions of Graphene & Pydantic
-graphene = ">=2.1.3,<3"
+graphene = ">=3.0b5"
```

Comment:
Since this represents a breaking change in the `graphene` versions supported, can we update the `version` tag for the overall project, to bump the minor version at least?

Reply:
I was unsure about what to update the version to. Will follow your suggestion and update it to 0.2.0.